src/web_interface/file_tree/file_tree.py

```diff
@@ -1,64 +1,101 @@
+from __future__ import annotations
+
 from itertools import chain
+from pathlib import Path
 from typing import Dict, Iterable, List, NamedTuple, Optional, Set
 
 from web_interface.file_tree.file_tree_node import FileTreeNode
 
+WEB_BASE_PATH = Path(__file__).parent.parent
+ICON_PATH = WEB_BASE_PATH / 'static/file_icons/mimetypes'
+TYPE_TO_PATH = {p.stem: f'/{p.relative_to(WEB_BASE_PATH)}' for p in ICON_PATH.iterdir()}
+CRYPTO = 'application-certificate'
+CONFIG = 'text-x-makefile'
+TYPE_TO_PATH.update(
```

Comment:
I'm confused why exactly we need this.
Shouldn't a mime type map to one icon?

Reply:
see below